INSIGHTS-ONPREM.md

```diff
@@ -0,0 +1,34 @@
+# Insights OnPrem instructions for starting koku in development environment
+## Kafka
+We use the ros-ocp-backend kafka container. Start the ros-ocp-backend kafka service.
+
+## Build and start
+- copy .env.example to new .env file
```

Comment:
can we migrate the `.env.example` to be `.env` and avoid this operation?

Reply:
I am trying to do as little changes as possible. but I can add a .env file.